{{cookiecutter.project_name}}/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## HEAD
```

Comment:
Please, rename it to `unreleased` so it's compliant with the keep-a-changelog-format.